src/VideoRemise/VideoRemise/HelpPage.xaml

```diff
@@ -0,0 +1,144 @@
+﻿<Page
+    x:Class="VideoRemise.HelpPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:VideoRemise"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <ScrollViewer Height="Auto" VerticalScrollBarVisibility="Auto">
+        <StackPanel Orientation="Vertical">
+            <Button Content="Return to Application" Margin="0,10,0,0" Click="OnDone" 
+                    FontSize="14" HorizontalAlignment="Center"/>
+            <RichTextBlock Margin="5,5,5,5" FontSize="14">
+                <Paragraph FontSize="18" FontWeight="Bold">Overview</Paragraph>
+                <Paragraph>
+                    VideoRemise works with your scoring machine and up to three
+                cameras to provide instant replay for fencing matches. It is open source 
+                so you can modify it or add features yourself if you like. VideoRemise
+                lives on
+                    <Hyperlink NavigateUri="https://github.com/alan-geller/video-remise">
+                        GitHub</Hyperlink> .
+                </Paragraph>
+                <Paragraph FontSize="18" FontWeight="Bold" Margin="0,5,0,0">Display</Paragraph>
+                <Paragraph>
+                    The main display has a command bar on the top and the video display below that.
+                The left side of the command bar shows the current system status; immediately below
+                that, the names of the fencers in the current match will be displayed if they have
+                been entered.
+                </Paragraph>
+                <Paragraph Margin="0,5,0,0">
+                    If there are multiple cameras, the video display area will show video from each 
+                camera.
+                Double-clicking on the video from one camera will expand that video and hide the
+                others; double-clicking again will return to the multi-camera view.
+                </Paragraph>
+                <Paragraph Margin="0,5,0,0">
+                    The command bar buttons are:
+                </Paragraph>
+                <Paragraph Margin="20,0,0,0">
+                    <InlineUIContainer>
+                        <SymbolIcon Symbol="People"/>
```

Comment:
May want to add a name as well as an icon.

Reply:
Done.